pyvista/core/partitioned.py

```diff
@@ -252,3 +252,22 @@ def append(self, dataset):
         index = self.n_partitions
         self.n_partitions += 1
         self[index] = dataset
+
+
+class PartitionedDataSetCollection(_vtk.vtkPartitionedDataSetCollection, DataObject):
+    """Wrapper for the ``vtkPartitionedDataSetCollection`` class."""
+
+    _WRITERS: ClassVar[Dict[str, Type[_vtk.vtkSimplePointsWriter]]] = {
+        ".vtcd": _vtk.vtkCompositeDataWriter,
```

Comment:
@CharlesGueunet I am wondering about the filename extension for saving a `PartitionedDataSetCollection`. Is there a commonly used extension for this dataset?

Reply:
Hello @tkoyama010 ,

Usually, we use the "vtpc" extension to serialize partitioned data set collection through the `vtkXMLPartitionedDataSetCollectionWriter`.
Also, we use the "vtpd" extension for partitioned data set.

Reply:
Thanks!
```suggestion
        ".vtpc": _vtk.vtkCompositeDataWriter,
```

Reply:
If possible, we try to use XML writers instead of the legagcy one.
I am not familiar enough with pyvista though, it may be the right choice.